src/identity/contextResolver.ts

```diff
@@ -3,15 +3,15 @@ import { FronteggPermissions } from '../../index';
 export const contextResolver = (req) => {
   if (!req.user) {
     return {
-      tenantId: '',
+      tenantId: 'NOT_A_REAL_TENANT_ID',
```

Comment:
WITHOUT_TENANT_ID

Reply:
do you want me to change the name to "WITHOUT_TENANT_ID"? @aviadmizrachi